sandbox-templates/svelte-developer/e2b.Dockerfile

```
@@ -0,0 +1,25 @@
+# Use Node.js base image for Svelte
+FROM node:21-slim
+
+# Install necessary system dependencies
+RUN apt-get update && apt-get install -y curl && apt-get clean && rm -rf /var/lib/apt/lists/*
+
+# Set working directory
+WORKDIR /home/user
+
+# Create a new Svelte 5 app with TypeScript and Tailwind CSS
+RUN npm create vite@latest svelte-app -- --template svelte-ts
+RUN cd svelte-app && npm install
```

Hadolint (2.12.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)